app/models/admin.rb

```diff
@@ -0,0 +1,7 @@
+class Admin < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
+  validates	:email, uniqueness: true
```

Comment:
A note that uniqueness only applies to admin and that an address can also be used for a user account. When brought up, this led to a discussion about why we have an admin and user model. We agreed to bring this up for discussion with Jason (Technical Lead)